Vagrantfile

```diff
@@ -1,18 +1,10 @@
-$script = <<-SCRIPT
-sudo rpm -Uvh https://packages.microsoft.com/config/rhel/7/packages-microsoft-prod.rpm
-echo "Run update"
-sudo yum update
-echo "Install dotnet core"
-sudo yum -y install dotnet-sdk-2.2
-SCRIPT
 Vagrant.configure("2") do |config|
-  config.vm.box = "centos/7"
+  config.vm.box = "jeffnoxon/ubuntu-20.04-arm64"
```

Comment:
why use this box rather than the official ubuntu one?

Reply:
https://app.vagrantup.com/generic/boxes/ubuntu2004
https://app.vagrantup.com/generic/boxes/ubuntu2010

Reply:
Honestly never tried those, used the jeffnoxon bc it's lighter

Reply:
Let's try and use the official one wherever possible